content/using-atom/sections/github-package.md

```diff
@@ -10,6 +10,7 @@ The github package brings Git and GitHub integration right inside Atom.
 - [Branch](#branch)
 - [Stage](#stage)
 - [Discard](#discard-changes)
+- [Undo discard](#undo-discard)
```

Comment:
Maybe this doesn't need to stand on its own there – as it's merely part of the "Discard" functionality.